examples/js/loaders/GLTFLoader.js

```diff
@@ -188,7 +188,8 @@ THREE.GLTFLoader = ( function () {
 
 				path: path || this.path || '',
 				crossOrigin: this.crossOrigin,
-				manager: this.manager
+				manager: this.manager,
+				onProgress: onProgress
```

Comment:
Since `onProgress` is scoped to the parser instance, I think `onLoad` and `onError` should be too, for consistency. Could you change this so that `GLTFParser.prototype.parse` takes no arguments, and both callbacks are passed here instead?